draft-ietf-mls-protocol.md

```diff
@@ -1617,27 +1621,26 @@ struct {
 
 The sender of a Remove message generates it as as follows:
 
+* Blank the path from the removed leaf to the root node
```

Comment:
We should note here that the blanking should be temporary / undone after this computation.  The generation of messages should not change the sender's state.

Reply:
Also, you should go ahead and truncate here, which will save you unnecessary derivations.  So:

* Before: Encrypt / blank
* After: Blank / Truncate / Encrypt

Same change in the processing logic below.